package.json

```diff
@@ -17,17 +17,17 @@
   "scripts": {
     "fix": "npm-run-all --continue-on-error \"fix:*\"",
     "fix:dependency-version-consistency": "check-dependency-version-consistency --fix .",
-    "fix:eslint": "eslint \"**/*\" --fix",
-    "fix:markdownlint": "markdownlint --fix \"**/*\"",
-    "fix:prettier": "prettier --cache --write --ignore-unknown \"**/*\"",
+    "fix:eslint": "eslint --fix \"**\"",
```

Comment:
since we're simplifying, can't we use `eslint . --fix`, `prettier . --write`?

Reply:
Nice idea! Done in https://github.com/blockprotocol/blockprotocol/pull/651/commits/2df1b0876bf088eb526b2ddf46cc48fe94161b39 (kept options before `.`)